src/frontend/components/Chat/SharedChatComponents/InputBar/index.tsx

```diff
@@ -67,40 +81,48 @@ export const InputBar = ({
         plain
         size={isChatInput ? '16px' : '13px'}
         style={{
-          fontWeight: isChatInput ? 'normal' : 'bold',
           fontFamily: 'Roboto-Regular',
+          fontWeight: isChatInput ? 'normal' : 'bold',
         }}
         value={inputText}
       />
-      <Box
-        border={{
-          size: 'xsmall',
-          color: 'blue-chat',
-        }}
-        height={isChatInput ? '40px' : '25px'}
-        round="6px"
-        style={{
-          minWidth: isChatInput ? '40px' : '25px',
-        }}
-        width={isChatInput ? '40px' : '25px'}
-      >
-        <ButtonStyled
-          fill
-          icon={
-            <Box align="center" alignContent="center" justify="center">
+
+      <ButtonStyled
```

Comment:
As you know I wasn't a big fan of this component, it is not getting any better.
I think you should split this component in multiple one to have a cleaner code.